core/domain/user_services.py

```diff
@@ -340,12 +356,14 @@ def get_users_settings(user_ids):
             result.append(UserSettings(
                 feconf.SYSTEM_COMMITTER_ID,
                 email=feconf.SYSTEM_EMAIL_ADDRESS,
+                role=feconf.ROLE_SUPER_ADMIN,
                 username='admin',
                 last_agreed_to_terms=datetime.datetime.utcnow()
             ))
         elif model:
             result.append(UserSettings(
-                model.id, email=model.email, username=model.username,
+                model.id, email=model.email, role=model.role,
```

Comment:
Yes, I know.

I did it as it was done similarly for email. Should I change it?